distribution/artifact/artifact-repackage.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+in=$1
+out=$(pwd)/$2
+files_to_remove=$3
+
+dest=$(mktemp -d)
+
+if file --dereference $in | grep 'zip'; then
+   unzip -d "$dest" "$in" &>/dev/null
+else
+   tar -xzf "$in" -C "$dest" &>/dev/null
+fi
+
+f=("$dest"/*)
+mv "$dest"/*/* "$dest"
+rmdir "${f[@]}"
+cd $dest || exit 1
+
+rm -f "$files_to_remove"
```

Comment:
- I think we should be able to remove a folder.
- Isn't `grep -v` solution quite simple? Why do you think it's not reliable?

Reply:
@lriuui0x0 addressed in https://github.com/graknlabs/dependencies/pull/260/commits/ef0a58aef5b7d7cc381b3874ab0071cade543e0e